packages/openqasm/quri_parts/openqasm/circuit/__init__.py

```diff
@@ -43,9 +43,10 @@
     gate_names.Z: "z",
     gate_names.H: "h",
     gate_names.S: "s",
-    gate_names.Sdag: "sdg",
+    gate_names.SqrtX: "sx",
+    gate_names.Sdag: "sdag",
     gate_names.T: "t",
-    gate_names.Tdag: "tdg",
+    gate_names.Tdag: "tdag",
```

Comment:
It seems that the changes in #49 are reverted, I guess it is unintentional.

Reply:
Sorry. I made a mistake.
I reflect the contents of #49.